docs/manual/non-empty-checker.tex

```diff
@@ -11,6 +11,23 @@
 \<Deque.getFirst()>, \<Deque.removeFirst()>, \<Queue.remove()>, or
 \<Queue.element()>.
 
+Currently, the Non-Empty Checker operates as an auxiliary checker for the
+Optional Checker (\sectionpageref{optional-checker}).
```

Comment:
I don't think this description is accurate any more.  I think the default modality is "normal" non-emptiness checking.